test/unit/math/rev/mat/prob/normal_id_glm_lpdf_gpu_test.cpp

```diff
@@ -0,0 +1,243 @@
+#ifdef STAN_OPENCL
```

Comment:
Please move the file under `/opencl/rev/` You can omit the `_gpu_` then. Also, in general we use the `_cl` suffix for everything OpenCL related.